ghost/staff-service/lib/emails.js

```diff
@@ -0,0 +1,291 @@
+
+            const templateData = {
+                memberData,
+                siteTitle: this.settingsCache.get('title'),
+                siteUrl: this.urlUtils.getSiteUrl(),
+                siteDomain: this.siteDomain,
+                accentColor: this.settingsCache.get('accent_color'),
+                fromEmail: this.fromEmailAddress,
+                toEmail: to,
+                staffUrl: this.urlUtils.urlJoin(this.urlUtils.urlFor('admin', true), '#', `/settings/staff/${user.slug}`)
+            };
+
+            const {html, text} = await this.renderEmailTemplate('new-free-signup', templateData);
+
+            await this.sendMail({
+                to,
+                subject,
+                html,
+                text
+            });
+        }
+    }
+
+    async notifyPaidSubscriptionStarted({member, subscription, offer, tier}, options = {}) {
+        const users = await this.models.User.getEmailAlertUsers('paid-started', options);
+
+        for (const user of users) {
+            const to = user.email;
+            const memberData = this.getMemberData(member);
+
+            const subject = `💸 Paid subscription started: ${memberData.name}`;
+
+            const amount = this.getAmount(subscription?.plan_amount);
+            const formattedAmount = this.getFormattedAmount({currency: subscription?.plan_currency, amount});
+            const interval = subscription?.plan_interval || '';
+            const tierData = {
+                name: tier?.name || '',
+                details: `${formattedAmount}/${interval}`
+            };
+
+            const subscriptionData = {
+                startedOn: this.getFormattedDate(subscription.start_date)
+            };
+
+            let offerData = this.getOfferData(offer);
+
+            const templateData = {
+                memberData,
+                tierData,
+                offerData,
+                subscriptionData,
+                siteTitle: this.settingsCache.get('title'),
+                siteUrl: this.urlUtils.getSiteUrl(),
+                siteDomain: this.siteDomain,
+                accentColor: this.settingsCache.get('accent_color'),
+                fromEmail: this.fromEmailAddress,
+                toEmail: to,
+                staffUrl: this.urlUtils.urlJoin(this.urlUtils.urlFor('admin', true), '#', `/settings/staff/${user.slug}`)
+            };
+
+            const {html, text} = await this.renderEmailTemplate('new-paid-started', templateData);
+
+            await this.sendMail({
+                to,
+                subject,
+                html,
+                text
+            });
+        }
+    }
+
+    async notifyPaidSubscriptionCanceled({member, cancellationReason, tier, subscription}, options = {}) {
+        const users = await this.models.User.getEmailAlertUsers('paid-canceled', options);
+        const subscriptionPriceData = _.get(subscription, 'items.data[0].price');
```

Comment:
We support the optional chaining now which is preferred to using lodash!

Reply:
Also it would have been nice to have this method define an object shape it wants, and the call site can pass that in, that way this package isn't dependant on the shape of subscriptions.

We're most likely going to be changing that soon, and we'll now need to either update this package, or map to a very nested and Stripe specific data structure

Reply:
> it would have been nice to have this method define an object shape it wants, and the call site can pass that in, that way this package isn't dependant on the shape of subscriptions.

Yeah i agree, I noted cleaning this up because it doesn't make sense for the this package to know the internals of subscription object, will do that 👌